release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2786,13 +2786,39 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.15.54
+[id="ocp-4-15-54_{context}"]
+=== RHBA-2025:10306 - {product-title} 4.15.54 bug fix and security update
+
+Issued: 09 July 2025
+
+{product-title} release 4.15.54, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHBA-2025:10306[RHBA-2025:10306] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHBA-2025:10307[RHBA-2025:10307] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.54 --pullspecs
+----
+
+[id="ocp-4-15-54-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, proxy variables were not respected in Machine Config Daemon pods during in-place upgrades due to missing proxy configuration integration. As a consequence, failed upgrades occurred due to incorrect image pulls. With this release, proxy variables are respected in Machine Config Daemon pods during successful in-place upgrades. (link:https://issues.redhat.com/browse/OCPBUGS-57553[OCPBUGS-57553])
```

Comment:
```suggestion
* Previously, proxy variables were not respected in Machine Config Daemon pods during in-place upgrades because of missing proxy configuration integration. As a consequence, failed upgrades occurred because of incorrect image pulls. With this release, proxy variables are respected in Machine Config Daemon pods during successful in-place upgrades. (link:https://issues.redhat.com/browse/OCPBUGS-57553[OCPBUGS-57553])
```